packages/server/lib/controllers/runner.coffee

```diff
@@ -43,7 +43,9 @@ module.exports = {
     ## https://github.com/cypress-io/cypress/issues/4952
     base64Config = Buffer.from(JSON.stringify(config)).toString('base64')
 
-    res.render(runner.getPathToIndex(), {
+    runnerPath = process.env.CYPRESS_INTERNAL_RUNNER_PATH or runner.getPathToIndex()
```

Comment:
Simplest way I could figure out to simulate a browser that won't connect is rendering an html page that doesn't have `socket.io`.